legal-api/report-templates/template-parts/notice-of-articles/benefitCompanyStmt.html

```diff
@@ -1,20 +1,21 @@
-<div class="no-page-break">
-  {% if business.legalType == 'BEN' %}
-    <div class="section-title mt-4">Benefit Company Statement</div>
-    <div class="section-data mt-4">
-        This company is a benefit company and, as such, has purposes that include conducting its
-        business in a responsible and sustainable manner and promoting one or more public benefits.
+{% if header.reportType != 'alterationNotice' %}
```

Comment:
isn't this the same statement used in the incorporation, and should be common?

Reply:
Fixed.